src/realm/sync/subscriptions.cpp

```diff
@@ -906,4 +906,16 @@ bool SubscriptionStore::would_refresh(DB::version_type version) const noexcept
     return version < m_db->get_version_of_latest_snapshot();
 }
 
+void SubscriptionStore::flush()
+{
+    auto tr = m_db->start_write();
```

Comment:
I will need to cancel them with an OperationAborted error and add a test. Since none of this code is currently being called, I will add this in a separate PR.